examples/Test/Foreign.hs

```diff
@@ -36,16 +37,16 @@ listExampleTests :: TestTree
 listExampleTests =
   testGroup
     "list tests"
-    [ testProperty "List.toList . List.fromList = id" invertNonGCList,
-      testProperty "map id = id" mapIdNonGCList,
-      testProperty "memory freed post-exception" testExecptionOnMem
+    [ testPropertyNamed "List.toList . List.fromList = id" "invertNonGCList" invertNonGCList,
+      testPropertyNamed "map id = id" "mapIdNonGCList" mapIdNonGCList,
+      testPropertyNamed "memory freed post-exception" "testExecptionOnMem" testExecptionOnMem
```

Comment:
Maybe you want to fix the typo `testExecptionOnMem` to `testExceptionOnMem`while you are at it.

Reply:
You know, it was all search and replace :slightly_smiling_face: .

Thanks for noticing!